_rules/SC1-1-1-image-has-name.md

```diff
@@ -19,21 +19,21 @@ authors:
 
 ### Applicability
 
-The rule applies to any HTML element with the [semantic role][] of `image` that is [exposed to assistive technologies][].
+The rule applies to any HTML element with the [semantic role](#semantic-role) of `image` that is [exposed to assistive technologies](#exposed-to-assistive-technologies).
```

Comment:
this links don't work correctly, they load this rule and not the algorithm.

Reply:
@jkodu I can't remember what we said about this, does this need to get resolved now or will it resolve itself once this is published?

Reply:
@jkodu can you confirm the above please?